README.md

```diff
@@ -6,6 +8,26 @@ This project is to setup a portfolio of mobile version skeleton main page
 ## Built With
 * HTML and CSS
 
+## Live Demo
+
+[Live Demo Link] (https://mikelobi.github.io/Personal-Portifolio/)
+
+
```

Comment:
- [ ] This link seems to be broken, please make sure that the application is properly deployed. 
<img width="878" alt="Screenshot 2022-02-28 at 15 44 44" src="https://user-images.githubusercontent.com/77038610/155978056-db568f46-0f7d-46c8-84be-2b4e4d54da1f.png">